Cargo.toml

```diff
@@ -168,3 +168,8 @@ members = [
 no-default-features = true
 features = ["macros", "num-bigint", "num-complex", "hashbrown", "serde", "multiple-pymethods", "indexmap", "eyre"]
 rustdoc-args = ["--cfg", "docsrs"]
+
+[patch.crates-io]
+# Instant misspells emscripten_get_now by including a leading underscore.
+# https://github.com/sebcrozet/instant/pull/47
+instant = { git = 'https://github.com/hoodmane/instant/', branch= 'emscripten-no-leading-underscore' }
```

Comment:
Well it is a two character change and I need it anyways for cryptography.

Reply:
Yeah it seems that cryptography has `pyo3 = { version = "0.15.2" }`  which requires `parking_lot = "0.11.0"` which requires `instant = "0.1.9"`. Indeed parking lot 0.12 has dropped the requirement. I wonder if it would work for me to build cryptography with a dependency-override giving it parking lot 0.12.1.

Reply:
Ah :(

Yes, there's https://github.com/pyca/cryptography/pull/6935, but they're blocked there waiting until 3.6 usage has decayed to a level where they're willing to drop support.

Reply:
Maybe I can make a PR asking them to include that. Was there any reason that pyo3 v0.15 pinned parking lot 0.11?

Reply:
I think just was probably just released before 0.12 existed